### PR TITLE
fix: increase google avatar_url

### PR DESCRIPTION
**Pull request by Lennart01**
## Description
Increases the maximum length of the Google avatar URL, as it was causing login issues when the avatar URL was too long.

### CODE REVIEW

File: server/src/database/migrations/sql/13_increase_gauth_avatar_url.up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE IF EXISTS google_users ALTER COLUMN avatar_url TYPE varchar(2048);
```
Comment (CronJorian) [inline server/src/database/migrations/sql/13_increase_gauth_avatar_url.up.sql:1]: Can you add a newline at the end? :)

**Review by CronJorian — APPROVED** [review]
It'd be reasonable to check whether the other authentications also require a longer url.
Otherwise lgtm 🚀

File: server/src/database/migrations/sql/13_increase_gauth_avatar_url.down.sql
Comment (brandstetterm) [inline server/src/database/migrations/sql/13_increase_gauth_avatar_url.down.sql:1]: I actually don't know if that works. What happens if there are urls with a length greater than 256?

Comment (Lennart01) [reply server/src/database/migrations/sql/13_increase_gauth_avatar_url.down.sql:1]: I guess that Postgres would refuse the operation.
Technically we would need to remove the avatar URL before migrating back or not migrating back at all.

**Review by brandstetterm — APPROVED** [review]

**Pull request merged**